pallets/funding/src/functions.rs

```diff
@@ -1127,6 +1129,8 @@ impl<T: Config> Pallet<T> {
 				plmc_usd_price,
 				did.clone(),
 				metadata_bidder_ticket_size_bounds,
+				existing_bids_amount + perform_bid_calls,
+				total_bids_for_project + perform_bid_calls,
```

Comment:
Do we need safe/check math for this?

Reply:
Its probably not required as we limit the max bids far under the max value, but lets just do it to be absolutely safe!

Reply:
👍